.github/workflows/kind-cluster-custom-resources.yaml

```diff
@@ -141,6 +141,7 @@ jobs:
       - name: Run Custom resources Tests
         timeout-minutes: 5
         run: |
+          chmod +x ./test/e2e_test_policy_custom_resource.sh
```

Comment:
FYI, you should be able to make this change in git so that you don't have to do this here.